spec/boac/boac_user_role_advisor_spec.rb

```diff
@@ -186,6 +206,22 @@
       end
     end
 
+    context 'visiting Everyone\'s Groups' do
+
+      it 'sees only curated groups created by CoE advisors' do
+        expected = BOACUtils.get_everyone_curated_groups(BOACDepartments::COE).map(&:id).sort
+        visible = (@group_page.visible_everyone_groups.map &:id).sort
+        @group_page.wait_until(1, "Missing: #{expected - visible}. Unexpected: #{visible - expected}") { visible == expected }
+      end
+
+      it 'cannot hit a non-COE filtered cohort URL' do
```

Comment:
Same here; should this be 'curated group URL'?

Reply:
@lyttam Yes, thanks. Amended.